eZ/Publish/Core/Repository/TrashService.php

```diff
@@ -294,7 +301,8 @@ public function findTrashItems(Query $query)
 
             foreach ($query->sortClauses as $sortClause) {
                 if (!$sortClause instanceof SortClause) {
-                    throw new InvalidArgumentValue('query->sortClauses', 'only instances of SortClause class are allowed');
+                    throw new InvalidArgumentValue('query->sortClauses',
```

Comment:
Please, fix CS

Reply:
Actually, such changes shouldn't be in PR until they are necessary to fix a bug 😉